src/main/scala/com/typesafe/sbt/bintraybundle/BintrayBundle.scala

```diff
@@ -0,0 +1,43 @@
+package com.typesafe.sbt.bintraybundle
+
+import sbt._
+import Keys._
+import com.typesafe.sbt.bundle.SbtBundle, SbtBundle.autoImport._
+import com.typesafe.sbt.packager.universal.UniversalPlugin, UniversalPlugin.autoImport._
+import bintray.{ Bintray, BintrayPlugin }, BintrayPlugin.autoImport._
+
+/**
+ * Provides the machinery to publish bundles to bintray adhering to the urn:x-bundle scheme
+ * described here: https://github.com/typesafehub/conductr-cli/issues/72
+ */
+object BintrayBundle extends sbt.AutoPlugin {
+
+  override def `requires` = SbtBundle && BintrayPlugin
+  override def trigger = AllRequirements
+
+  override def projectSettings: Seq[Setting[_]] =
+    Seq(
+      bintrayReleaseOnPublish in Bundle := false,
+      bintrayRepository in Bundle := "bundle",
+      publishMavenStyle in Bundle := false
+    ) ++ settings(Bundle)
+
+  def settings(config: Configuration): Seq[Setting[_]] =
+    inConfig(config)(Seq(
+      bintrayPackage := (normalizedName in config).value,
+      publish := {
+        val packageName = (bintrayPackage in config).value
+        val repo = Bintray.cachedRepo(
+          (bintrayEnsureCredentials in config).value,
+          (bintrayOrganization in config).value,
+          (bintrayRepository in config).value
+        )
+        val bundleDist = (dist in config).value
+        val version = bundleDist.getName.split("-").toList.takeRight(2).mkString("-").replaceAll("""\.zip$""", "")
+        val bundleExt = ".zip"
+        val path = s"/${repo.owner}/$packageName/$version/$packageName-$version$bundleExt"
```

Comment:
@huntc - looking back at the urn convention for bundle in https://github.com/typesafehub/conductr-cli/issues/72 - is `${repo.owner}` actually `repository` not `organisation`?